demo/assets/Airplane/Airplane.gltf

```diff
@@ -34,7 +34,8 @@
     {
       "mesh": 0,
       "name": "11805_airplane_v2_L2",
-      "rotation": [0.7071068286895752, 0, 0, 0.7071067094802856]
+      "rotation": [0.7071068286895752, 0, 0, 0.7071067094802856],
+      "scale": [0.005, 0.005, 0.005]
```

Comment:
The algorithm doesn't take this into account, so far. In order to prevent triangle clouds scale should be configured within the original asset (in blender)

Reply:
When i scale it in blender, it does also just add a scale transform here.